tests/Features/homepage.feature

```diff
@@ -0,0 +1,6 @@
+Feature: Homepage
+
+    Scenario: It loads!
+        Given I am on the homepage
+        Then I should see "OpenCFP"
```

Comment:
I don't think we should test for seeing OpenCFP but rather something else that we can use to identify the page.

Since users updating OpenCFP for their own conference will get a failing test suite

Reply:
We could change it to something other than OpenCFP, but my personal opinion is that this is *our* test suite dependent on values from *our* `testing.yml`. This "feature" is arguably a candidate for removal from the suite, in my opinion.